.travis.yml

```diff
@@ -1,12 +1,9 @@
-dist: trusty
+dist: xenial
 language: python
 
 matrix:
     include:
-        # Explicitly tell Travis which Python version to use for which
-        # tox environment, instead of using one Python version in Travis and
-        # hoping that tox and pyenv run the tests in the desired versions.
-        # https://github.com/travis-ci/travis-ci/issues/8363#issuecomment-355090242
+        # Run each tox env in a travis env that sure has the required Python
```

Comment:
Should this be:

`Run each tox env in a travis env that has the required Python verison`

Reply:
Fixed in 7fd8c0032342c7049058b5795ba571e5132b2e8a. Let me know if that sounds better.